feat(wasm): Add initial docs for WASM (#2737)
getsentry/sentry-docs: src/platforms/native/common/debug-information.mdx

Settings > Processing Issues_, which shows a list of all the necessary files and
instructions to retrieve them.

> <PlatformSection notSupported={["native.wasm"]}>

Reviewer: Add empty lines (also at the closing tag):

```suggestion
<PlatformSection notSupported={["native.wasm"]}>

```

Otherwise it doesn't parse as Markdown and renders like this:

![image](https://user-images.githubusercontent.com/1433023/100853675-53240c00-3488-11eb-9739-e86a93b3e12c.png)

In addition to Debug Information Files, Sentry needs _Call Frame Information_
(CFI) to extract accurate stack traces from minidumps of optimized release
builds. CFI is usually part of the executables and not copied to debug symbols.
Unless you are uploading Breakpad symbols, be sure to also include the binaries
when uploading files to Sentry.
</PlatformSection>

<PlatformSection supported={["native.wasm"]}>

> For WASM you need to provide both the debug information and the original code
> in the same debug information file.  For this purpose we provide a custom
> tool called [wasm-split](https://github.com/getsentry/symbolicator/tree/master/wasm-split)
> which can do the splitting for you:

Reviewer 2: ```suggestion
For WASM, you need to provide both the debug information and the original code
in the same debug information file.  To do this, we provide a custom
tool called [wasm-split](https://github.com/getsentry/symbolicator/tree/master/wasm-split)
that can do the splitting for you:
```

```shell
wasm-split /path/to/file.wasm -d /path/to/file.debug.wasm --strip
```

> This modifies the `file.wasm` in place to add the `build_id` and then removes all
> debug information, but puts that into a `file.debug.wasm` instead.  That latter file
> then needs to be uploaded to Sentry.

Reviewer 2: ```suggestion
This command modifies the `file.wasm` in place to add the `build_id`, then removes all
debug information. The debug information is put in a `file.debug.wasm` which 
then needs to be uploaded to Sentry.
```
